worlds/stardew_valley/logic.py

```diff
@@ -779,100 +1004,187 @@ def has_junimo_kart_power_level(self, power_level: int) -> StardewRule:
             return True_()
         return self.received("Junimo Kart: Extra Life", power_level)
 
+    def has_junimo_kart_max_level(self) -> StardewRule:
```

Comment:
With ```full_shuffling``` this will return True based solely on the number of Extra Lives you have, without taking into account whether you can actually access Junimo Kart. That might be intended, given the function name, in which case ```lets_play_a_game``` needs the access requirement